### DISCUSSION TITLE
add slug capabilities to v9?

### DISCUSSION THREAD

**Discussion by sumberacstefano** [directus/directus#12416] [Archive]
Hi team,

https://www.npmjs.com/package/directus-extension-wpslug-interface
Found this extension. Extension works great! 

But why were slug capabilities even removed from Directus? You had some issues or what was the reason?
Would be beautiful to have them again in next release without adding any extensions. 

bye👋

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/4231 🙂
